app/(explore)/get-funded/page.tsx

```diff
@@ -97,16 +97,17 @@ export default function Career() {
                                         <h2 className="text-base lg:text-xl font-semibold font-montserrat">
                                             Common Application
                                         </h2>
-                                        <div
-                                            role="button"
-                                            tabIndex={0}
+                                        <a
+                                            href="https://grants.bitcoindevs.xyz/"
+                                            target="_blank"
+                                            rel="noopener noreferrer"
```

Comment:
```suggestion
                                            rel="noopener strict-origin"
```

Now that I'm in here I'm wondering if we want to send the BDP website as the referrer. Any objections?